packages/common/src/search/_internal/portalSearchItems.ts

```diff
@@ -173,7 +173,7 @@ async function searchPortalAsHubSearchResult(
   const fn = (item: IItem) => {
     return itemToSearchResult(
       item,
-      searchOptions.includes,
```

Comment:
This typo was preventing the `include` list from working... downside of allowing `searchOptions` to have `[key: string]: any` as an outlet is that we also lost checks